### DISCUSSION TITLE
Custom default site?

### DISCUSSION THREAD

**Discussion by Lombra** [nginx-proxy/nginx-proxy#1808] [Q&A] [answered]
Hi,

Is there a way to utilise a custom default site directly within nginx-proxy without running a whole other nginx container?

**Comment by tkw1536** [comment, marked as answer]
The intended way is to use a 'DEFAULT_HOST' as described in https://github.com/nginx-proxy/nginx-proxy#default-host. 

This requires a dedicated container, but you don't have to use a full blown nginx if all you want is a static site. You could e.g. use https://github.com/PierreZ/goStatic.
